dev/com.ibm.ws.event.logging_fat/fat/src/com/ibm/ws/event/logging/FATTest.java

```diff
@@ -680,6 +680,7 @@ public void testEventLoggingLogModeUpdate3() throws Exception {
     @Test
     public void testEventLoggingLogModeUpdate2() throws Exception {
         server.setServerConfigurationFile("server_logModeExit.xml");
+        server.waitForStringInLog("CWWKG0017I", 90000);
```

Comment:
Is "CWWKG0017I" mandatory?  If yes, please surround with `assertNotNull()`

Reply:
Updated PR